Lm01/solution

danman01 opened the pull request:
Updating solution branch

raq929 commented on index.html:
```diff
@@ -16,7 +16,7 @@
           <input type="text" name="book[id]" id="book-id" placeholder="Resource ID Here">
         </div>
-        <button type="submit" class="btn btn-default">Get all Books</button>
+        <button type="submit" class="btn btn-default">Submit</button>
```
So, I think the point here is to add to the app's functionality, not overwrite what we did before. This should be done as a new form, so that they now have both index and show on their app.

raq929 commented:
I think my comment on this should be fixed before merge. These lessons are meant to be cumulative, so at the end of the sequence, you have full CRUD on a resource.

danman01 commented:
@raq929 I updated the solution branch to have two forms. Updated submit handlers to accept either form. Also updated the index.html to have titles to explain what's going on when each form is submitted.
